src/BoundingBox.h

```diff
@@ -72,7 +72,6 @@ class CBoundingBox
 	 */
 	Vec3f getMaxPoint(void) const { return m_maxPoint; }
 	
-private:
```

Comment:
because I was using one of the private variables in another class and it was not accessible. I know I could have used static casting or get method, but this was giving me an error at submission